packages/app/features/components/settingsListItem.jsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Flex, H3, Image, Text, View } from 'dripsy'
+import React from 'react'
+
+export const SettingsListItem = ({
+  title = 'Pose #1',
+  info,
+  style,
+}) => {
+  return (
+    <>
+        <View sx={{ height: 8 }} />
```

Comment:
I might not know where this component is going to be used, but I think you might be able to add a marginTop here instead of adding a new div